devtools/gradle/gradle-application-plugin/src/main/java/io/quarkus/gradle/tasks/worker/BuildWorker.java

```diff
@@ -72,12 +91,44 @@ public void execute() {
                     LOGGER.info("    .jar.uberJar = {}", jar.isUberJar());
                 }
             }
-
             LOGGER.info("Quarkus application build was successful");
         } catch (BootstrapException e) {
             // Gradle "abbreviates" the stacktrace to something human-readable, but here the underlying cause might
             // get lost in the error output, so add 'e' to the message.
             throw new GradleException("Failed to build Quarkus application for " + gav + " due to " + e, e);
         }
     }
+
+    private static class Slf4JMessageWriter implements MessageWriter {
+        private final Logger LOGGER;
```

Comment:
The case is weird, it should be lowercase.

Reply:
yup... copy/paste error

Reply:
On a follow up PR